usagov/usagov-2021
# USAGOV-1787-wizard-field-configs-update: Update wizard field configs based on new requirements.

## Jira Task


https://cm-jira.usa.gov/browse/USAGOV-1787

## Description

Enable html text format in wizard body and footer fields. Show options list views reference field in form display.

## Type of Changes

  - Add screenshot showing what it should look like
- [ X ] Drupal Config (requires "drush cim")

## Testing Instructions


1. Edit an existing wizard page.
2. At the bottom of the form, in the select menu for display, change the selection to "select". 
3. Delete the view from the options list autocomplete selection.
4. Save.
5. View the page. There is no options list.
6. Edit the page and add the options list view to the autocomplete form, and select block to display, just as it was before.
7. Save and view the page.
8. The options list appears as expected.
9. Edit the page and change the text format for the body and footer fields to "html". 
10. Write some html in these fields.
11. Save and view the page.
12. The html you wrote in step 10 appears as expected.

### Change Requirements



## Security Review




## Reviewer Reminders

- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions

Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
13. Find the commit of this pull request.
14. Build and deploy the changes.
15. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
